pages/index.md

```diff
@@ -3,7 +3,7 @@
 You're using it right now! The website you're reading is hosted on my home
 computer, through boringproxy.
 
-boringproxy is a combination reverse proxy and tunnel manager.
+`boringproxy` is a combination of reverse proxy and tunnel manager.
```

Comment:
combination of reverse -> combination of a reverse

Reply:
"a reverse" also invited for writing "a tunnel"